lib/install/management/dispatcher.go

```diff
@@ -181,8 +196,8 @@ func (d *Dispatcher) Dispatch(conf *metadata.VirtualContainerHostConfigSpec, set
 		return err
 	}
 
-	if err = d.removeApplianceIfForced(conf); err != nil {
-		return errors.Errorf("%s", err)
+	if err = d.checkExistence(conf); err != nil {
```

Comment:
We should change the name of Dispatch to Create or Deploy - doesn't need to be done in this PR, but at the moment it's single purpose with a general purpose name which is confusing